flight/Modules/SimVisualization/simvisualization.c

```diff
@@ -77,7 +77,7 @@ int32_t SimVisualizationStart(void)
 	return 0;
 }
 
-MODULE_INITCALL(VisualizationInitialize, VisualizationStart)
+MODULE_INITCALL(SimVisualizationInitialize, SimVisualizationStart)
```

Comment:
Yes.  Before on the simulator the MODULE_INITCALLs weren't used and modules were "manually" invoked.  This is fixed in this PR, so it needs valid symbol names.